src/common/helper.js

```diff
@@ -543,20 +552,29 @@ function * checkGetAccess (authUser, submission, parentSpan) {
         const screener = _.filter(currUserRoles, { role: 'Primary Screener' })
         const reviewer = _.filter(currUserRoles, { role: 'Reviewer' })
 
-        // User is either a Reviewer or Screener
-        if (screener.length !== 0 || reviewer.length !== 0) {
-          const screeningPhase = _.filter(phases, { phaseType: 'Screening', phaseStatus: 'Scheduled' })
+        if (screener.length !== 0) { // User is Screener
+          const screeningPhase = _.union(
+            _.filter(phases, { phaseType: 'Screening', phaseStatus: 'Open' }),
+            _.filter(phases, { phaseType: 'Screening', phaseStatus: 'Closed' })
+          )
+
+          // Screening is not Opened / Closed
+          if (screeningPhase.length !== 0) {
```

Comment:
@meshde With this logic, if there is a phase where screening phase is open or closed, then screener cannot access the submission...

The expectation is that if the contest has a screening phase AND the phase is NOT in status Scheduled (it is either open or closed), then the screener can access the submission... Let me know if I am wrong here though - the flowchart is a bit complex so I could be in the wrong here too...

Reply:
Oh yes, this was left out by oversight. Will fix it right away.

Reply:
@callmekatootie The code has been updated. 

In the current implementation, the screener will not be able to access the submission in cases when the challenge has no screening phase at all. Is this fine or should the code be updated to provide access in such a case?

Reply:
No that's fine. If there is no Screening phase, then a role of Screener is irrelevant...